src/amo/sagas/reviews.js

```diff
@@ -235,9 +253,25 @@ function* manageAddonReview(
     );
 
     yield put(setReview(reviewFromResponse));
+
+    if (savingRating) {
+      yield put(flashReviewMessage(SAVED_RATING));
+    }
+    if (savingReview) {
+      yield put(flashReviewMessage(SAVED_REVIEW));
```

Comment:
Is there any flickering if both `savingRating` and `savingReview` are `true`?

Reply:
In this UI it won't be possible to trigger both at the same time. If it comes up, we could handle it pretty easily by accepting both messages.